pkg/blob/nodeserver.go

```diff
@@ -213,12 +225,35 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	klog.V(2).Infof("target %v\nprotocol %v\n\nvolumeId %v\ncontext %v\nmountflags %v\nmountOptions %v\nargs %v\nserverAddress %v",
 		targetPath, protocol, volumeID, attrib, mountFlags, mountOptions, args, serverAddress)
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
 
 	authEnv = append(authEnv, "AZURE_STORAGE_ACCOUNT="+accountName, "AZURE_STORAGE_BLOB_ENDPOINT="+serverAddress)
-	cmd.Env = append(os.Environ(), authEnv...)
+	var output []byte
+	var resp *mount_azure_blob.MountAzureBlobResponse
+	// create a context with 5 secs timeout.
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	klog.V(2).Infof("Dailing GRPC server with 5 seconds timeout")
```

Comment:
remove this logging since there is already consecutive logging whether success or failure